src/api/controllers/articleController.js

```diff
@@ -92,17 +92,27 @@ export default class ArticleController {
     const articles = await Article.findAndCountAll({
```

Comment:
I made changes in this file because as I created a separate table for tags, I had to redo the way tags were added in the database for an article